internal/metamorph/server.go

```diff
@@ -208,11 +211,14 @@ func (s *Server) PutTransactions(ctx context.Context, req *metamorph_api.Transac
 		timeout = txReq.GetMaxTimeout()
 
 		// Convert gRPC req to store.StoreData struct...
+		callback := store.StoreCallback{
+			CallbackURL:   txReq.GetCallbackUrl(),
+			CallbackToken: txReq.GetCallbackToken(),
+		}
 		sReq := &store.StoreData{
 			Hash:              hash,
 			Status:            statusReceived,
-			CallbackUrl:       txReq.GetCallbackUrl(),
-			CallbackToken:     txReq.GetCallbackToken(),
+			Callbacks:         []store.StoreCallback{callback},
```

Comment:
Same thing here, if you think it's a better way

Reply:
Done